core/src/main/scala/tofu/package.scala

```diff
@@ -3,13 +3,15 @@ import cats.{ApplicativeError, MonadError}
 import cats.data.Ior
 
 package object tofu {
+  type In[C, F[_]] = WithContext[F, C]
```

Comment:
What's the use?

Reply:
Just small type alias to write `X In F` instead of `F WithContext X`